lib/engine/game/g_18_ny/step/discard_train.rb

```diff
@@ -10,6 +10,10 @@ class DiscardTrain < Engine::Step::DiscardTrain
           def process_discard_train(action)
             @game.salvage_train(action.train)
           end
+
+          def description
+            'Salvage Excess Trains'
```

Comment:
The rulebook explicitly uses `discard` here.

```
Whenever trains are salvaged or discarded, the owning company receives the salvage value
```

```
If a phase change includes a reduction in the Train Limit [3], all companies, including the one
operating, must immediately discard any trains in excess of the Train Limit to the Bank Pool 
for salvage value.
```

Reply:
It does, but is that the optimum phrasing?

Reply:
Since the site doesn't add the text "for salvage value", I feel this is an improvement.

Reply:
Salvage, as a verb, means to save something from destruction. So salvaging a train would be rescuing one from the scrap yard and putting it back into use, the opposite of what is happening here.

Reply:
How about `discard for salvage value`?